sportsdataverse/nhl/nhl_loaders.py

```
@@ -1,5 +1,7 @@
-# GENERATED by tools/codegen/generate.py -- DO NOT EDIT.
+# GENERATED by tools/codegen/generate.py -- DO NOT EDIT (generated section).
```

Ruff (0.15.15)

[warning] The file is executable but no shebang is present

(EXE002)